flat/BaseRootTunnel.sol

```diff
@@ -1670,9 +1663,7 @@ abstract contract BaseRootTunnel is AccessControlMixin {
         stateSender.syncState(childTunnel, message);
     }
 
-    function _validateAndExtractMessage(bytes memory inputData) internal returns (bytes memory) {
-        require(inputData.length == 10, "RootTunnel: BAD_PAYLOAD");
```

Comment:
Why isn't this require needed anymore?

Reply:
Check https://github.com/maticnetwork/pos-portal/pull/96 @wschwab

Reply:
@itzmeanjan pointed to #96 to explain this